iothub/service/src/Jobs/Models/ScheduledJob.cs

```diff
@@ -36,13 +36,6 @@ internal JobType AlternateJobType
             set => JobType = value;
         }
 
-        /// <summary>
```

Comment:
Should CraetedOnUtc also be pulled into the base class? It is part of the response, no?